tests/torch/sota_checkpoints_eval.json

```diff
@@ -470,7 +503,8 @@
         "mapillary_vistas": {
             "unet_mapillary": {
                 "config": "examples/torch/semantic_segmentation/configs/unet_mapillary.json",
-                "target": 56.24,
+                "target_ov": 56.24,
+                "target_pt": 56.24,
```

Comment:
On practic for OV it's not equal, and more for PT that tested on GPU we get different metric build by build. 
So separated `target` and `target_diff` for PT and OV is good solution for this test.

Reply:
Metric for PT is expected to differ from run to run, but also expected to be minor, stay in a given range (which is why we set up a threshold of `0.1`) and be distributed around a given value (which we call `target`). If OV runtime does not reproduce this PT value, then there is a bug somewhere, because OV is expected to do inference functionally the same as the framework (with only minor discrepancies). So the target for OV is defined exclusively by the target for PT (it is equal to it) in the normal case, and the normal case should only specify `"target_ov"` if there is something unfixable about OV metrics differing from PT metrics. Right now only SSD models would be the candidates for specifying `"target_ov"`, other models have OV metrics conformant to the PT metric and it should stay so.

Reply:
Ofc, it can be easly done by `target_pt = sample_dict.get("target_pt", sample_dict["target_ov"]`.
But differ will be only that reference will be less on several lines, and there is no other improvements.
OV and PT_CUDA metrics is not equal in general case, so i have no see any problem to have two targets in reference file.

So looks like 'target_ov' should be prioritized metrics for test because:
- OV provides stable results 
- we can expect 0.0 in 'Target Diff' for all models 
- not clear which metrics value should be used for target_pt in case of unstable result (in case where i update metric for pt, i used mean for several runs).
- and final argument: it's e2e test and the final step is validation OV model as target inference backend.

So, i can keep current state or i can use `target_ov` as default value for `target_pt` and add `target_pt` only for ssd models?


@AlexKoff88  @alexsu52  what do think about it?

Reply:
Honestly, I don't see any difference in these solutions, both apply. Historically, we always use only target metric from framework to calculate accuracy drop. @KodiaqQ what is your opinion?

Reply:
In the ideal scenario, we should have only one column that we should rely on - `reference fp32`, and calculate a 1% `absolute` drop (if applicable) for all cases: OV, PT, TF, no matter. In this scenario, it's valid to extend the drop threshold in individual cases (like SSD), but it should be calculated on the fly in general.
This is my opinion, and the final decision should be made by a majority vote.